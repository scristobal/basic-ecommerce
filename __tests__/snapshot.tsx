import { render } from '@testing-library/react';
import Home from '../src/app/page';
import { it, expect } from '@jest/globals';

it('renders homepage unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
});
