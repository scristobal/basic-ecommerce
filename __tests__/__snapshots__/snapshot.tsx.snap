// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <nav>
    <h1
      class="text-6xl"
    >
      Homepage
    </h1>
    <a
      href="/about"
    >
      About
    </a>
  </nav>
</div>
`;
